docs/sanic/logging.md

```diff
@@ -9,17 +9,53 @@ A simple example using default settings would be like this:
 
 ```python
 from sanic import Sanic
+from sanic.log import logger
+from sanic.response import text
 
 app = Sanic('test')
 
 @app.route('/')
 async def test(request):
-    return response.text('Hello World!')
+    logger.info('Here is your log')
+    return text('Hello World!')
 
 if __name__ == "__main__":
   app.run(debug=True, access_log=True)
 ```
 
+After the server is running, you can see some messages looks like:
```

Comment:
Can you take this logo part out of the documentation?

Reply:
@ahopkins Sure, no problem. But is that mean the `debug` option in `app.run()` also needed to modify? Or there may some user will be confused why there is a logo show up in logs.

Reply:
I mean there is no reason to add it into the docs right now because we likely will be removing it in the future. I think the likelihood of confusion from it being in someone's logs but not in the documentation is low.